python/libraries/SystemIO.py

```diff
@@ -8,6 +10,15 @@ async def inp(question):
     with ThreadPoolExecutor(1, "AsyncInput") as executor:
         return await asyncio.get_event_loop().run_in_executor(executor, input, question)
 
+def run(command):
+    can_run = os.environ.get("COMMAND_ALLOW") == "true"
+    if not can_run:
+        return False
+
+    return os.system(COMMAND_ALLOW) == 0
```

Comment:
Is `COMMAND_ALLOW` defined?

Reply:
Its an environment variable, just like `N_WS_DEBUG`

Reply:
Hmm so is there a difference between `COMMAND_ALLOW` and `os.environ.get('COMMAND_ALLOW')`?

Reply:
Also because `command` is unused and `COMMAND_ALLOW` must be `true` at that point, so `os.system('true')` will almost always return 1 (an error)

Reply:
OH WAIT, that should be `command`, oops